src/dashboard/src/templates/administration/sidebar.html

```diff
@@ -48,7 +49,11 @@
       <li class="{% active request url_user_list %}"><a href="{{ url_user_list }}">{% trans "Users" %}</a></li>
     {% endif %}
 
-    <li class="{% active request url_language %}"><a href="{{ url_language }}">{% trans "Language" %}</a></li>
+    <li class="{% active request handle_config %}"
+        ><a href="{{ handle_config }}">{% trans "Handle server config" %}</a></li>
```

Comment:
Haha. Sorry, that's just me. Everything I write has a 79col rule! (emails, notes, html) I know it's crazy. I have line-wrapping turned off in my editor, which is why you see this kind of stuff.

But there's another problem here, no? It looks like I removed the "Language" link!